source/Ship.h

```diff
@@ -225,8 +232,15 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	
 	// Check if the ship is thrusting. If so, the engine sound should be played.
 	bool IsThrusting() const;
+	bool IsReversing() const;
+	bool IsSteering() const;
+	// The direction that the ship is steering. If true, the ship is steering right. 
+	// If false, the ship is steering left.
+	bool SteeringDirection() const;
```

Comment:
To me it would make sense if we just mirrored the current `Turn` command setting, where -1 is left and +1 is right, or use a scoped enum.

Mimicking `Turn` could let us differentiate fractional thrust, while the scoped enum would limit comparison logic by consumers.

Reply:
SteeringDirection is now a double equal to commands.Turn().